recipe/meta.yaml

```diff
@@ -35,18 +33,45 @@ test:
     - python
     - unittest2
```

Comment:
I see the comments in the code which suggest this _could_ run under py3... would it be worth doing so? 8 months left...